variables.tf

```diff
@@ -24,6 +24,10 @@ variable "allocated_storage" {
   description = "The allocated storage in GBs."
   default     = 10
 }
+variable "max_allocated_storage" {
+  description = "When configured, the upper limit to which Amazon RDS can automatically scale the storage of the DB instance. See the terraform documentation for more information."
```

Comment:
@CorinChappy If we are going to say "see terraform docs" it would be good to include a link somewhere.

Reply:
There's nothing that says you can use markdown i the variable descriptions, happy to try it though. Otherwise we can just leave it in the readme